_std/defines/speech_defines/chat_prefixes.dm

```diff
@@ -0,0 +1,20 @@
+#define CHAT_PREFIX_RADIO ":"
+#define CHAT_PREFIX_RADIO_GENERAL ";"
+
+#define CHAT_PREFIX_AI_RADIO_1 ":1"
+#define CHAT_PREFIX_AI_RADIO_2 ":2"
+#define CHAT_PREFIX_AI_RADIO_3 ":3"
+#define CHAT_PREFIX_AI_RADIO_DEFAULT ":"
+#define CHAT_PREFIX_AI_RADIO_GENERAL ";"
```

Comment:
Should these not just use the two above? It seem confusing to have two defines that resolve to the same prefix.